### ISSUE TITLE
singularity

### ISSUE THREAD

**Issue by schraderL** [fansalon/TEspeX#6]
Hi! 
This looks like a great tool for TE expression quantification. The installation is a bit messy on a HPC environment, though. Do you plan to provide a singularity or docker container to make TEspeX more accessible?

cheers
lukas

**Comment by fansalon** [comment]
Hi Lukas,

no, there is no plan to provide any TEspeX singularity/docker container. Docker still requires sudo privileges, so I am not sure it might be the way to go. Singularity might be slightly better, but as said, there is nothing planned on this side.

I am, however, surprised you are experiencing issues in installing it on a HPC environment. TEspeX installation guide is mostly based on conda "operations", which should - in theory - work on HPC. 

Nevertheless, I am happy to help on the installation so if you tell me which steps you are experiencing issues in, we can work on this together to make it working on your HPC system and improve the README.

Thanks!

All the best,
Federico

**Comment by schraderL** [comment]
Hi Federico
thank you for the quick reply! I got stuck at zlib, because `$LDFLAGS` and `$CFLAGS` were not defined for me, but `$LD_LIBRARY_PATH` was. 
In the end, I don't think I needed to set these, because the test runs fine without them.  

In general, running conda in an HPC environment can be annoying, because every user need to install software for themselves and I can't install it for my students for example. Also, the HPC admins can't easily install software that depends on conda as a module. 

For this containers are just a nice solution. I honestly don't know how hard it is to set up a container. 

I will see when I find time to test TEspeX further, but I'm sure I'll come back here with more questions. :) 

Cheers
Lukas
